OctopusDSC/DSCResources/cOctopusServer/cOctopusServer.psm1

```diff
@@ -249,7 +249,7 @@ function Test-OctopusVersionSupportsAutoLoginEnabled {
     return Test-OctopusVersionNewerThan (New-Object System.Version 3, 5, 0)
 }
 
-function Test-OctopusVersionSupportsHsts {
+function Test-OctopusVersionSupportsHst {
```

Comment:
I think I'd prefer the (excessively verbose) name `Test-OctopusVersionSupportsHttpStrictTransportSecurity` rather than dropping that `s`. Either that, or maybe a suppress on that rule (similar to [this one](https://github.com/mikejwhat/OctopusDSC/blob/9d13b21a5dd6e2fff97480925b449dc86fb276f2/OctopusDSC/DSCResources/cOctopusServer/cOctopusServer.psm1#L296))?